### PR TITLE
configure: read kernel header location from environment variable

### PR DESCRIPTION
**Pull request by Mic92**
otherwise use "/usr/include" as default

### CODE REVIEW

**Comment by andreamattavelli** [comment]
@Mic92 Looks good to me. Just wondering if `KERNEL_HEADERS` is somehow a standard environment variable name.

File: configure [hunk 1]
```diff
@@ -181,7 +181,7 @@ def findKernelIncludePath():
         which are needed to build uclibc
     """
 
-    std_include = "/usr/include"
+    std_include = os.environ.get("UCLIBC_KERNEL_HEADERS", "/usr/include")
```
Comment (Mic92) [inline configure:184]: Now it should be safe.

**Comment by Mic92** [comment]
ping.

**Comment by ccadar** [comment]
LGTM too, let's merge this.

**Pull request merged**